II.pass-tests/11-cities-formatter/cities.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function formatCities(capitals, transform) {
+  return capitals.map(capital => transform(capital));
```

Comment:
Note you can simplify here: `return capitals.map(transform);`. `transform` is already a function that takes the current array item as the first parameter and returns the result for the new array, so you can use it directly.